src/components/mdx/link.js

```diff
@@ -23,3 +23,32 @@ export const Link = React.forwardRef(function CustomLink(props, ref) {
     </ChakraLink>
   )
 })
+
+export const Doi = (props) => {
+  const doi = props.doi
+  if (doi) {
+    return (
+      <Link href={`https://doi.org/${doi}`}>
+        <Image src={`https://zenodo.org/badge/DOI/${doi}.svg`} alt='DOI' />
+      </Link>
+    )
+  }
+}
+
+export const ShowcaseTitle = (props) => {
+  const url = props.url
+  const title = props.title
+  if (url) {
+    return (
+      <Link href={url} fontSize={'xl'} fontWeight={'bold'}>
+        {title}
+      </Link>
+    )
+  } else {
+    return (
+      <Text fontSize={'xl'} fontWeight={'bold'}>
+        {title}
+      </Text>
+    )
+  }
+}
```

Comment:
@Shane98c @andersy005, is there any chance one of you would be willing to do me a favor by reviewing this section of the code? I have negative confidence in my JavaScript skills 😅